tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -890,7 +998,7 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
                         invocationsAllowed += 1
                         actionResult = Some(JsObject(
                             "headers" -> JsObject(
-                                "content-type" -> "xyz/bar".toJson),
+                                "content-type" -> "xyzbar".toJson),
```

Comment:
> Is this related to the removal of getForKey above?

yes - `getForKey()` tests that the media type is known to Akka-Http which isn't very useful there are a lot of media types are not in their list. Hence I changed `findContentTypeInHeader` to only do the `parse()` which ensures that it the format is valid, but not do the check against the list Akka-Http knows about.